tests/integ/modin/window/test_rolling.py

```diff
@@ -212,8 +235,11 @@ def test_rolling_window_unsupported():
             pd.Timestamp("20130101 09:00:06"),
         ],
     )
-    with pytest.raises(NotImplementedError):
-        snow_df.rolling(window="2s", min_periods=None).sum()
+    with pytest.raises(
+        NotImplementedError,
+        match="'center=True' is not implemented with str window for Rolling.sum",
+    ):
+        snow_df.rolling(window="2s", min_periods=1, center=True).sum()
```

Comment:
nit: remove `min_periods=1` for clarity?

Reply:
Will do, pandas apparently defaults `min_periods` to 1 is window is a time string/offset so fixed that as well.